docs/permissions.rst

```diff
@@ -471,31 +497,113 @@ access privileges than its unauthenticated counterpart in case of contradictory
 doesn't remain denied access due to its implicit ``MAGPIE_ANONYMOUS_GROUP`` membership. Step (2.3) handles this edge
 case specifically.
 
-Every *generic* :term:`Group` share the same priority, and will therefore resolve conflicting :class:`Access` using the
-normal step conditions and prioritizing :attr:`Access.DENY`.
+Every *generic* :term:`Group` (i.e.: others than ``MAGPIE_ANONYMOUS_GROUP`` and ``MAGPIE_ADMIN_GROUP``) share the same
+priority, and will therefore resolve conflicting :class:`Access` using the normal step conditions and prioritizing
+:attr:`Access.DENY` (e.g.: step (2.2)).
 
 When resolving only :term:`Inherited Permissions`, the procedure stops here and provides the applicable result if any
 was found, with the corresponding ``reason``. An empty set of :term:`Permission` is returned if none could be found.
 
-When instead resolving :term:`Effective Permissions`, the above process continues by rewinding the parent
-:term:`Resource` hierarchy until the first :term:`Permission` is found. Only on the first iteration (when the targeted
-:term:`Resource` is the same as the one looked for potential :term:`Inherited Permissions`) does :attr:`Scope.MATCH`
-take effect. Only :attr:`Scope.RECURSIVE` are considered afterwards. When a :term:`Permission` is found, the process
-immediately completes if :attr:`Access.DENY` results from the previous resolution steps. Otherwise, the process still
-continues until reaching the top-most :term:`Service` to validate :attr:`Access.ALLOW` over the entire scope the
-:term:`Permission` was found. If still no :term:`Permission` is defined after complete hierarchy processing, the result
-defaults to :attr:`Access.DENY`, and indicated by ``"no-permission"`` reason.
-
-.. todo: pseudo-code to represent the 'effective permission' portion of the resolution
-
+When instead resolving :term:`Effective Permissions`, there is an additional pre-step to (1) that verifies if
```

Comment:
I love the new documentation. I'm just wondering why you didn't respect the chronological order (admin as step 1 and rewinding as a step, so that we get 4 steps : 

1. admin
2. direct
3. inherited
4. rewinding

Reply:
I can adjust it with these steps to make it clearer.

Reply:
And this is the same for the rewinding step, I did not catch this particularity. Maybe you could do something like that : 
1. [effective only] admin
2. [all type] direct
3. [inherited and effective] inherited
4. [effective only] rewinding